README.md

```diff
@@ -11,60 +11,74 @@ Extender is a build server that builds native extensions of the Defold engine. T
+As result you should have 2 jars in `./server/apps/` folder: extender.jar and manifestmergetool.jar.
 
+### Run
+To run stand-alone Extender instance use folowing script:
 ```sh
-$ /usr/local/extender/current/service.sh stop /usr/local/extender
+    ./server/scripts/standalone/service-standalone.sh start
 ```
+Script takes commad as 1st argument. Command can be one of the following:
+* **start** - start new Extender instance
+* **stop** - stop already running Extender instance
+* **restart** - stop and start Extender instance from the scratch
 
-### Developing using a stand-alone server on macOS
-
-Use the `rebuild-and-run-standalone-local.sh` script to quickly rebuild and launch a new version of the stand-alone server:
-
+If you want to use different Spring profile you can pass it via 2nd argument. For example,
 ```sh
-$ ./server/scripts/rebuild-and-run-standalone-local.sh /path/to/localextender
+    ./server/scripts/standalone/service-standalone.sh start standalone-dev
 ```
 
-This will set the `TARGET_DIR` environment variable to `/path/to/localextender`, stop any currently running server, build a new one, deploy and start it and show the server log in the console.
+Note: all Spring profiles should be located in `./server/configs/` folder.
+
+### Develop/debug standalone Extender using VSCode
+Note that [Prerequisites](#prerequisites) should be completed and manifestmergetool.jar already downlaoded or built.
+
+1. Download VSCode: https://code.visualstudio.com/download
+2. Install following extensions:
+   1. Spring Boot Extension Pack https://marketplace.visualstudio.com/items?itemName=vmware.vscode-boot-dev-pack
+   2. Spring Boot Dashboard https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-spring-boot-dashboard
+   3. Language Support for Java(TM) by Red Hat https://marketplace.visualstudio.com/items?itemName=redhat.java
+   4. Gradle for Java https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-gradle
+   5. Debugger for Java https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-debug. Note: version should be v0.58.2024090204 (pre-release) or greater.
+3. Open folder with Extender sources in VSCode.
+4. Go to `Run and Debug` and open `launch.json` 
+
+   ![](images/run_debug.png)
+
+5. Add following launch configuration
+    ```json
+        {
+            "type": "java",
+            "name": "Spring Boot-ExtenderApplication<server>",
+            "request": "launch",
+            "cwd": "${workspaceFolder}",
+            "mainClass": "com.defold.extender.ExtenderApplication",
+            "projectName": "server",
+            "args": "--spring.config.location=classpath:./,file:${workspaceFolder}/server/configs/ --extender.sdk.location=${workspaceFolder}/server/app/sdk --spring.profiles.active=standalone-dev",
+            "envFile": ["${workspaceFolder}/server/envs/.env", "${workspaceFolder}/server/envs/user.env", "${workspaceFolder}/server/envs/macos.env"]
```

Comment:
I guess this part could be an actual .vscode project?

Reply:
Also, it there's a way to setup the dependencies in the project as well, it'd be even easier to use 🤔 

Seems there might exist something like that?
https://docs.lextudio.com/blog/different-ways-to-create-vscode-extension-dependencies/#extension-dependencies-in-configuration